database_wrapper.py

```diff
@@ -377,48 +426,54 @@ def save_audio_sample_meta_data(self, formatted_data: dict) -> bool:
         # create an AudioSampleMetaData object with the given metadata
         metadata = AudioSampleMetaData()
 
-        isWW = formatted_data['isWakeWord']
-        if ((isWW == 'ww') or (isWW is True)):
+        isWW = formatted_data["isWakeWord"]
+        if (isWW == "ww") or (isWW is True):
             metadata.is_wake_word = True
-        elif ((isWW == 'nww') or (isWW is False)):
+        elif (isWW == "nww") or (isWW is False):
             metadata.is_wake_word = False
         else:
             msg = "unexpected values for isWakeWord\n"
             msg += "expected 'ww' or True or 'nww' or False but got '{}'"
-            msg = msg.format(formatted_data['isWakeWord'])
+            msg = msg.format(formatted_data["isWakeWord"])
             raise BadDictionaryValueError(msg)
 
-        metadata.first_name = formatted_data['firstName']
-        metadata.last_name = formatted_data['lastName']
-        metadata.gender = formatted_data['gender']
+        metadata.first_name = formatted_data["firstName"]
+        metadata.last_name = formatted_data["lastName"]
+        metadata.gender = formatted_data["gender"]
 
-        if (formatted_data['noiseLevel'] == 'q' or
-                formatted_data['noiseLevel'] == 'quiet'):
+        if (
+            formatted_data["noiseLevel"] == "q"
+            or formatted_data["noiseLevel"] == "quiet"
+        ):
             metadata.noise_level = NoiseLevel.quiet
-        elif (formatted_data['noiseLevel'] == 'm' or
-              formatted_data['noiseLevel'] == 'medium'):
+        elif (
+            formatted_data["noiseLevel"] == "m"
+            or formatted_data["noiseLevel"] == "medium"
+        ):
             metadata.noise_level = NoiseLevel.medium
-        elif (formatted_data['noiseLevel'] == 'l' or
-              formatted_data['noiseLevel'] == 'loud'):
+        elif (
+            formatted_data["noiseLevel"] == "l"
+            or formatted_data["noiseLevel"] == "loud"
+        ):
```

Comment:
I'm not sure if I like the multiline if statements - it seems like it takes up a lot of vertical space (I know this was the linter - maybe we should change some settings in the future? I don't think this should block the PR.)

Reply:
https://black.readthedocs.io